packages/eslint-plugin/src/rules/pseudo-element-naming.ts

```diff
@@ -0,0 +1,80 @@
+import { ESLintUtils, TSESTree } from '@typescript-eslint/utils';
+
+import { createRule } from '../utils/createRule';
+import { isStringLiteral, isMakeStylesIdentifier, isObjectExpression, isProperty } from '../utils/helpers';
+
+export const RULE_NAME = 'pseudo-element-naming';
+
+const UNSUPPORTED_PSEUDO_ELEMENTS: Record<string, true> = {
```

Comment:
```suggestion
const PSEUDO_ELEMENTS = [':before', ':after']
```

nit: let's just use `PSEUDO_ELEMENTS` as name and list items as an array.